src/components/Tutorial/TutorialSwap/TutorialMobile.tsx

```diff
@@ -0,0 +1,56 @@
+import { Trans } from '@lingui/macro'
+import React from 'react'
+import { X } from 'react-feather'
+import { Flex, Text } from 'rebass'
+import styled from 'styled-components'
+
+import ToggleCollapse, { ToggleItemType } from 'components/Collapse'
+import { Z_INDEXS } from 'constants/styles'
+import useTheme from 'hooks/useTheme'
+
+import { TOTAL_STEP } from './constant'
+
+const Wrapper = styled.div`
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: ${Z_INDEXS.MOBILE_MODAL};
+  position: fixed;
+  background: ${({ theme }) => theme.buttonGray};
+`
+export default function TutorialMobile({
+  stopTutorial,
+  steps,
+  isOpen,
+}: {
+  isOpen: boolean
+  stopTutorial: () => void
+  steps: ToggleItemType[]
+}) {
+  const theme = useTheme()
+  return (
+    <Wrapper>
+      <Flex flexDirection="column" alignItems="center" style={{ overflowY: 'scroll', height: '100%', width: '100%' }}>
+        <div style={{ padding: 16, marginBottom: 3, width: '100%' }}>
+          <Flex justifyContent="space-between" marginBottom="10px">
+            <Text fontSize={16} fontWeight={500} color={theme.text}>
+              <Trans>Welcome to KyberSwap!</Trans>
+            </Text>
+            <X color={theme.subText} size={24} onClick={stopTutorial} />
+          </Flex>
+          <Text fontSize={12} color={theme.subText}>
+            {TOTAL_STEP} <Trans>easy ways to get started with KyberSwap</Trans>
```

Comment:
move `{TOTAL_STEP}` inside Trans

Reply:
just a number, we can skip translate it.

Reply:
You wont know in other language it will translated into corresponding order

Reply:
It might be translated into `text text text {TOTAL_STEP} text text text` in some language